src/Icons/Models/IconLink.cs

```diff
@@ -23,12 +23,15 @@ public class IconLink
     private const string _jpegMediaType = "image/jpeg";
     private static readonly byte[] _jpegHeader = new byte[] { 255, 216, 255 };
 
+    private const string _svgXmlMediaType = "image/svg+xml";
```

Comment:
💭 Does there need to be a code expansion below to check the header? I see this done for the other types.

Reply:
I don't think we need to, those are only for formats that we try to infer if we don't get a `Media-Type` header back or we don't support the media type we got back. I also don't think there are generally "marker" bytes for svg.

Reply:
Throwing in an additional test theory with a site that uses an SVG icon would be a nice and quick addition.

Reply:
there doesn't need to be mime type detection on the file level. That's just a fallback if the mimetype doesn't exist.

I don't believe svg+xml has a mime type file header (I couldn't find one) so it's not added.